apps/core/views.py

```diff
@@ -293,3 +295,33 @@ def post(self, request, *args, **kwargs):
         user.save()
 
         return Response('Your password has been changed')
+
+
+class PasswordChangeView(APIView):
+    queryset = User.objects.all()
+    permission_classes = (IsAuthenticated,)
+    serializer_class = PasswordChangeSerializer
+
+    def get_serializer(self, *args, **kwargs):
+        return self.serializer_class(self, *args, **kwargs)
```

Comment:
This function here looks like a redundant

Reply:
Without get_serializer:
<img width="568" alt="image" src="https://github.com/jakkemomo/meetups/assets/111009132/1e659408-aded-4cfc-8649-d8dfaade42a6">
With it:
<img width="516" alt="image" src="https://github.com/jakkemomo/meetups/assets/111009132/06261828-2b5c-49a2-a3cb-eccd3e21ffa6">

Reply:
Let's add a comment that this function is needed/used to generate swagger documents